.travis.yml

```diff
@@ -12,8 +22,11 @@ install:
   - chmod +x /tmp/urchin/package/urchin
   - '[ -z "$WITHOUT_CURL" ] || sudo apt-get remove curl -y'
 script:
+  - export CXX=g++-4.8
+  - export CC=gcc-4.8
```

Comment:
Then I'd have to define it once for each matrix item, instead of once for the entire build.